package.json

```diff
@@ -11,29 +11,24 @@
     "test-watch": "mocha --watch",
     "predocs": "node scripts/write-features.js",
     "docs": "docco --output docs src/*.js",
-    "build": "babel --presets es2015 --out-dir dist src",
```

Comment:
That means that babel and related presets are not longer needed in the `devDependencies`, right?

Reply:
Yes. I removed babel from devDependencies.

Reply:
👍